integration/compliance_aws_test.go

```diff
@@ -158,3 +158,12 @@ func TestComplianceAwsGetReportRecommendationID(t *testing.T) {
 	assert.Contains(t, out.String(), "AFFECTED RESOURCES",
 		"STDOUT table headers changed, please check")
 }
+
+func TestComplianceAwsSearch(t *testing.T) {
+	out, err, exitcode := LaceworkCLIWithTOMLConfig(
+		"compliance", "aws", "search", "example",
+	)
+	assert.Empty(t, err.String(), "STDERR should be empty")
+	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
+	assert.Contains(t, out.String(), "No results found.", "STDOUT changed, please check")
+}
```

Comment:
can we add a successful integration test? that is, search for a dummy resource that actually exists.

Reply:
maybe a silly S3 bucket

Reply:
yes!